jsonrpc/src/test.rs

```diff
@@ -52,10 +54,9 @@ where
     };
 
     // create tokio futures runtime
```

Comment:
I think this comment should be removed too

Reply:
thx Jan fixed